--- init/init.c	2019-07-13 05:20:57.034805787 +0000
+++ init/init.c	2019-07-13 05:19:19.421645633 +0000
@@ -174,7 +174,7 @@
 #define ONCE        0x04
 /*
  * NB: while SYSINIT/WAIT/ONCE are being processed,
- * SIGHUP ("reread /System/Settings/busybox/inittab") will be processed only after
+ * SIGHUP ("reread /etc/inittab") will be processed only after
  * each group of actions. If new inittab adds, say, a SYSINIT action,
  * it will not be run, since init is already "past SYSINIT stage".
  */
@@ -670,14 +670,14 @@
  * then parse_inittab() simply adds in some default
  * actions (i.e., runs INIT_SCRIPT and then starts a pair
  * of "askfirst" shells).  If CONFIG_FEATURE_USE_INITTAB
- * _is_ defined, but /System/Settings/busybox/inittab is missing, this
+ * _is_ defined, but /etc/inittab is missing, this
  * results in the same set of default behaviors.
  */
 static void parse_inittab(void)
 {
 #if ENABLE_FEATURE_USE_INITTAB
 	char *token[4];
-	parser_t *parser = config_open2("/System/Settings/busybox/inittab", fopen_for_read);
+	parser_t *parser = config_open2("/etc/inittab", fopen_for_read);
 
 	if (parser == NULL)
 #endif
@@ -930,7 +930,7 @@
 {
 	struct init_action *a, **nextp;
 
-	message(L_LOG, "reloading /System/Settings/busybox/inittab");
+	message(L_LOG, "reloading /etc/inittab");
 
 	/* Disable old entries */
 	for (a = G.init_action_list; a; a = a->next)
@@ -1196,7 +1196,7 @@
 			+ (1 << SIGTERM) /* reboot */
 			+ (1 << SIGUSR2) /* poweroff */
 #if ENABLE_FEATURE_USE_INITTAB
-			+ (1 << SIGHUP)  /* reread /System/Settings/busybox/inittab */
+			+ (1 << SIGHUP)  /* reread /etc/inittab */
 #endif
 			, record_signo);
 
@@ -1268,10 +1268,10 @@
 //usage:#define init_full_usage "\n\n"
 //usage:       "Init is the first process started during boot. It never exits."
 //usage:	IF_FEATURE_USE_INITTAB(
-//usage:   "\n""It (re)spawns children according to /System/Settings/busybox/inittab."
+//usage:   "\n""It (re)spawns children according to /etc/inittab."
 //usage:	)
 //usage:	IF_NOT_FEATURE_USE_INITTAB(
-//usage:   "\n""This version of init doesn't use /System/Settings/busybox/inittab,"
+//usage:   "\n""This version of init doesn't use /etc/inittab,"
 //usage:   "\n""has fixed set of processed to run."
 //usage:	)
 //usage:
@@ -1279,7 +1279,7 @@
 //usage:	"This version of init is designed to be run only by the kernel.\n"
 //usage:	"\n"
 //usage:	"BusyBox init doesn't support multiple runlevels. The runlevels field of\n"
-//usage:	"the /System/Settings/busybox/inittab file is completely ignored by BusyBox init. If you want\n"
+//usage:	"the /etc/inittab file is completely ignored by BusyBox init. If you want\n"
 //usage:	"runlevels, use sysvinit.\n"
 //usage:	"\n"
 //usage:	"BusyBox init works just fine without an inittab. If no inittab is found,\n"
@@ -1295,7 +1295,7 @@
 //usage:	"	tty3::askfirst:/bin/sh\n"
 //usage:	"	tty4::askfirst:/bin/sh\n"
 //usage:	"\n"
-//usage:	"If you choose to use an /System/Settings/busybox/inittab file, the inittab entry format is as follows:\n"
+//usage:	"If you choose to use an /etc/inittab file, the inittab entry format is as follows:\n"
 //usage:	"\n"
 //usage:	"	<id>:<runlevels>:<action>:<process>\n"
 //usage:	"\n"
@@ -1352,13 +1352,13 @@
 //usage:	"\n"
 //usage:	"		Unrecognized actions (like initdefault) will cause init to emit an\n"
 //usage:	"		error message, and then go along with its business. All actions are\n"
-//usage:	"		run in the order they appear in /System/Settings/busybox/inittab.\n"
+//usage:	"		run in the order they appear in /etc/inittab.\n"
 //usage:	"\n"
 //usage:	"	<process>:\n"
 //usage:	"\n"
 //usage:	"		Specifies the process to be executed and its command line.\n"
 //usage:	"\n"
-//usage:	"Example /System/Settings/busybox/inittab file:\n"
+//usage:	"Example /etc/inittab file:\n"
 //usage:	"\n"
 //usage:	"	# This is run first except when booting in single-user mode\n"
 //usage:	"	#\n"
--- include/usage.h	2019-07-13 05:27:24.514911047 +0000
+++ include/usage.h	2019-07-13 05:27:02.410648802 +0000
@@ -2401,10 +2401,10 @@
 #define init_full_usage "\n\n" \
        "Init is the first process started during boot. It never exits." \
 	IF_FEATURE_USE_INITTAB( \
-   "\n""It (re)spawns children according to /System/Settings/busybox/inittab." \
+   "\n""It (re)spawns children according to /etc/inittab." \
 	) \
 	IF_NOT_FEATURE_USE_INITTAB( \
-   "\n""This version of init doesn't use /System/Settings/busybox/inittab," \
+   "\n""This version of init doesn't use /etc/inittab," \
    "\n""has fixed set of processed to run." \
 	) \
 
@@ -2412,7 +2412,7 @@
 	"This version of init is designed to be run only by the kernel.\n" \
 	"\n" \
 	"BusyBox init doesn't support multiple runlevels. The runlevels field of\n" \
-	"the /System/Settings/busybox/inittab file is completely ignored by BusyBox init. If you want\n" \
+	"the /etc/inittab file is completely ignored by BusyBox init. If you want\n" \
 	"runlevels, use sysvinit.\n" \
 	"\n" \
 	"BusyBox init works just fine without an inittab. If no inittab is found,\n" \
@@ -2428,7 +2428,7 @@
 	"	tty3::askfirst:/bin/sh\n" \
 	"	tty4::askfirst:/bin/sh\n" \
 	"\n" \
-	"If you choose to use an /System/Settings/busybox/inittab file, the inittab entry format is as follows:\n" \
+	"If you choose to use an /etc/inittab file, the inittab entry format is as follows:\n" \
 	"\n" \
 	"	<id>:<runlevels>:<action>:<process>\n" \
 	"\n" \
@@ -2485,13 +2485,13 @@
 	"\n" \
 	"		Unrecognized actions (like initdefault) will cause init to emit an\n" \
 	"		error message, and then go along with its business. All actions are\n" \
-	"		run in the order they appear in /System/Settings/busybox/inittab.\n" \
+	"		run in the order they appear in /etc/inittab.\n" \
 	"\n" \
 	"	<process>:\n" \
 	"\n" \
 	"		Specifies the process to be executed and its command line.\n" \
 	"\n" \
-	"Example /System/Settings/busybox/inittab file:\n" \
+	"Example /etc/inittab file:\n" \
 	"\n" \
 	"	# This is run first except when booting in single-user mode\n" \
 	"	#\n" \
@@ -5368,7 +5368,7 @@
 
 #define cttyhack_full_usage "\n\n" \
        "Give PROG a controlling tty if possible." \
-     "\nExample for /System/Settings/busybox/inittab (for busybox init):" \
+     "\nExample for /etc/inittab (for busybox init):" \
      "\n	::respawn:/bin/cttyhack /bin/sh" \
      "\nGiving controlling tty to shell running with PID 1:" \
      "\n	$ exec cttyhack sh" \
